object/virtual_machine.go

```diff
@@ -263,7 +263,7 @@ func (v VirtualMachine) WaitForNetIP(ctx context.Context, v4 bool, device ...str
 	p := property.DefaultCollector(v.c)
 
 	// Wait for all NICs to have a MacAddress, which may not be generated yet.
-	err := property.Wait(ctx, p, v.Reference(), []string{"config.hardware.device"}, func(pc []types.PropertyChange) bool {
+	_ = property.Wait(ctx, p, v.Reference(), []string{"config.hardware.device"}, func(pc []types.PropertyChange) bool {
```

Comment:
Looks like this error should be returned rather than ignored.

Reply:
@dougm I did it, and it brokes the test suite. See [here](https://travis-ci.org/vmware/govmomi/jobs/506161754).  Should I TAL at the tests?

Reply:
@mjtrangoni do you want to try again (returning err here) and see if CI is ok with the change?